docs/source/release/v6.7.0/Diffraction/Powder/New_features/35284.rst

```diff
@@ -0,0 +1 @@
+- It is now possible to provide more than one detector height range in :ref:`ILLPowderDetectorScan`
```

Comment:
Could you also mention that two properties have been removed from this algorithm?

Reply:
Done